### PR TITLE
Multiple values per modbus hvac mode

### PR DESCRIPTION
**Pull request by escoand**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/98829
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -613,7 +613,8 @@ climates:
           required: true
           type: integer
         write_registers:
-          description: "Request type, use `write_registers` if true  else `write_register`."
+          description: "Request type, use `write_registers` if true  else `write_register`.
+            If more than one value is specified for a specific mode, only the first one is used for writing to the register."
```
Comment (janiversen) [inline source/_integrations/modbus.markdown:617]: Which mode, there are no explanation of how to set multiple modes or multiple values.

**Review by janiversen — CHANGES_REQUESTED** [review]
I think you miss the essential of your change, that multiple values can be defined.

File: source/_integrations/modbus.markdown (hunk 1 above)
Comment (escoand) [reply source/_integrations/modbus.markdown:617]: Ahh sorry, copy and paste...

**Review by janiversen — APPROVED** [review]
LGTM, thanks.

**Review by janiversen — APPROVED** [review]
LGTM, thanks (Thought I did that already)

**Pull request merged**